src/logic/colorMod.ts

```diff
@@ -0,0 +1,28 @@
+import { LocalStorageKeys } from '../utils/types';
+
+const colorModParams = {
+  icon: false,
+  bg: false,
+};
+
+export const colorMod = (e: MouseEvent): void => {
+  const colorModBtn = document.querySelector('.actions__bg') as HTMLElement;
+  const html = document.querySelector('html') as HTMLElement;
+
+  if (e.target === colorModBtn) {
+    colorModParams.icon = colorModBtn.classList.toggle('white');
+    colorModParams.bg = html.classList.toggle('dark');
+    localStorage.setItem(LocalStorageKeys.bg, JSON.stringify(colorModParams));
+  }
+};
+
+export const getColorMod = (): void => {
+  const colorModBtn = document.querySelector('.actions__bg') as HTMLElement;
+  const html = document.querySelector('html') as HTMLElement;
+
+  const getColorModParam = JSON.parse(localStorage.getItem(LocalStorageKeys.bg) || JSON.stringify(colorModParams));
+  if (getColorModParam.icon === true && getColorModParam.bg === true ) {
```

Comment:
No need to compare with true 
Just getColorModParam.icon && ......
And why icon is boolean value 
Maybe isIcon ?